rasa/cli/utils.py

```diff
@@ -225,13 +226,16 @@ def validate_files(
         max_history: The max history to use when validating the story structure.
         importer: The `TrainingDataImporter` to use to load the training data.
         stories_only: If `True`, only the story structure is validated.
+        flows_only: If `True`, only the flows are validated.
     """
     from rasa.validator import Validator
 
     validator = Validator.from_importer(importer)
 
     if stories_only:
         all_good = _validate_story_structure(validator, max_history, fail_on_warnings)
+    elif flows_only:
+        all_good = validator.verify_flows()
```

Comment:
Indeed, no user warnings are issued. I was acting on feedback from CSE that this `rasa data validate` behaviour doesn't make sense for users who have asked for errors to be raised because it's easier to understand where the issue is: see https://rasahq.atlassian.net/browse/ATO-1549. While the plan was not to refactor the whole validation code,  I kept this change restricted to flows, then we can plan for other changes. I can raise this with the squad to gather more opinions and was planning to add in the docs how validation of flows will be different to the old `rasa data validate`.

Reply:
Addressed the discussion conclusion we had on Slack on proceeding with logging errors: https://github.com/RasaHQ/rasa/pull/12866/commits/e529e07c2d3307afedf95a9d10c14437e2b4e359